dev-python/pydotplus/pydotplus-2.0.2.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```

Comment:
it's only listed under python 2.7 and 3.4 [on PyPI](https://pypi.python.org/pypi/pydotplus).